includes/admin/class-wc-rest-stripe-settings-controller.php

```diff
@@ -646,6 +646,13 @@ private function update_enabled_payment_methods( WP_REST_Request $request ) {
 				}
 			}
 
+			// Cards are handled by the main gateway so we store a separate option rather than updating the global gateway's enabled setting directly.
+			if ( in_array( 'card', $payment_method_ids_to_enable, true ) ) {
+				$this->gateway->update_option( 'cards_enabled', 'yes' );
+			} else {
+				$this->gateway->update_option( 'cards_enabled', 'no' );
+			}
```

Comment:
Because we don't have a separate CC gateway class or seperate gateway settings for credit cards, like we do for other APMS, I've used a new option stored on the main gateway class that records if cards are enabled or not.